### ISSUE TITLE
how use script convert all en datasets to the format 'conll'

### ISSUE THREAD

**Issue by Crescentz** [juand-r/entity-recognition-datasets#3]
Sorry, is there a demo to show how to use your scripts?

**Comment by juand-r** [comment]
No, there isn't a demo unfortunately.  I need to revist those scripts at some point so they are easier to use. Is there a particular dataset you want to convert to 'conll'?
